mycroft/skills/diagnostics/__init__.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2016 Mycroft AI, Inc.
+#
+# This file is part of Mycroft Core.
+#
+# Mycroft Core is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Mycroft Core is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Mycroft Core.  If not, see <http://www.gnu.org/licenses/>.
+
+from os.path import dirname
+
+from adapt.intent import IntentBuilder
+from mycroft.skills.core import MycroftSkill
+from mycroft.util.log import getLogger
+import psutil
+import subprocess
+import json
+from time import time
+
+__author__ = 'the7erm'
+
+LOGGER = getLogger(__name__)
+
+
+def and_(strings):
+    """
+    Join a list of strings with , and add 'and' at the end, because grammar
+    matters.
+    """
+    if len(strings) <= 1:
+        return " ".join(strings)
+
+    return "%s and %s" % (", ".join(strings[0:-1]),
```

Comment:
This method will create a language dependency for this skill, which will be hard to maintain. Let's **not** rely on the `and` term and keep the `,` (comma separation) only. That said, we can kill this method and use the `join` only, such as: `", ".join(ips)` and without worrying if your `ips` list has one or more elements. If you have only one element, the comma will not be added.